### PR TITLE
Add error message for missing webserver

### PR DESCRIPTION
**Pull request by krauselukas**
I wasn't able to clone a job because the apache2 service was disabled on my machine, so i added a message at this place.

### CODE REVIEW

File: script/clone_job.pl [hunk 1]
```diff
@@ -301,7 +301,7 @@ sub clone_job {
         }
     }
     else {
-        die "failed to create job: ", pp($tx->res->body);
+        die "failed to create job: please make sure your http proxy is running (apache, nginx, etc.) ", pp($tx->res->body);
```
Comment (okurz) [inline script/clone_job.pl]: * After ":" start with capital letter
* I think you can limit it to the error code you got. Was it 304?

Comment (krauselukas) [inline script/clone_job.pl]: @okurz I just got the line where the issue occured ("failed to create job: "" at ./script/clone_job.pl line 304.")

Comment (okurz) [inline script/clone_job.pl]: ah, I see. So you at least know that the string `pp($tx->res->body)` was empty so you could do

```perl
else {
    die "Failed to create job, empty response. Make sure your HTTP proxy is running, e.g. apache, nginx, etc." unless $tx->res->body;
    die "Failed to create job: ", pp($tx->res->body);
}
```

Comment (krauselukas) [inline script/clone_job.pl]: @okurz allright, thanks :)

**Comment by okurz** [comment]
LGTM

**Pull request merged**